styleguide/derek/_partials/_rug.ejs

```diff
@@ -1,12 +1,47 @@
 <!--[rug]-->
-<div class="container rug">
+<div class="container-fluid bg-blue-gray half-padding">
+<div class="container hidden-xs hidden-sm">
   <div class="row">
     <div class="col-sm-3">
-      <div class="col-sm-3">
-        <h5>We've got your back</h5>
-        <h4 class="text34">24x7x365</h4>
+      <h5 class="rug-subhead-medium no-margin">We've got your back</h5>
+      <h4 class="rug-subhead-large no-margin">24<small class="white">x</small>7<small class="white">x</small>365</h4>
     </div>
+    <div class="col-sm-3">
+      <div class="row">
+        <div class="col-lg-2 col-md-3">
+          <i class="fa fa-mobile fa-4x rug-icon"></i>
```

Comment:
Applying a class making it clear this is an icon in the rug helps identify it for both CSS and JS (if needed). The `.rug-icon` class has a `color: $white` and applying the class changes the color. No super selector (`.rug *`) is needed to color this element.